index.html

```diff
@@ -17,12 +17,18 @@
     <header>
         <!-- nav bar -->
         <div>
-            <nav>
-                <ul>
-                    <li class="logo"><a href="index.html">Chuckster</a></li>
-                    <li class="nav-links" id="breadcrumb"></li>
-                </ul>
-                <ul class="nav-links">
+            <nav class="navbar">
+                <div class="logo-and-menu-btn-container">
+                    <ul>
+                        <li class="logo"><a href="index.html">Chuckster</a></li>
+                        <li class="nav-links" id="breadcrumb"></li>
+                    </ul>
+                    <a href="javascript:void(0);" class="mobile-menu" onclick="responsiveMenu()">
```

Comment:
We've been using "#" as the href to not redirect in all the rest of the anchor tags.

Also, we've been using eventlisteners in the JS instead on onclick html functions to keep the JS separate from the HTML

Reply:
Done